### PR TITLE
Update contributing/_index.en.md

### PR DESCRIPTION
**Pull request by nyechiel**
Getting some up to date content on this page.

### CODE REVIEW

File: src/content/contributing/_index.en.md [hunk 1]
```diff
@@ -5,17 +5,23 @@ weight = 150
...
 
-### Proposing a pull request
-Once your code is ready to be reviewed, you can propose a pull request.
+#### [#submariner](https://kubernetes.slack.com/archives/C010RJV694M)
+Share your ideas in the #submariner channel in Kubernetes' Slack. If you need it, you can [request an invite to K8S slack instance](https://slack.k8s.io/).
+
+#### [Community Calendar](https://calendar.google.com/calendar/r?cid=NHFuZGVoOGY0bzZ1ajlvZnBsczh1NWNlZ2tAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+As a member of the Submariner Community, join any of our community meetings - no registration required. The weekly [Submariner Community Meeting](https://tinyurl.com/wfbx37q) (Tuesdays at 5:00pm CET) is a good place to start.
+
+
```
Comment (dfarrell07) [inline src/content/contributing/_index.en.md:22]:
```suggestion
```

File: src/content/contributing/_index.en.md [hunk 2]
```diff
@@ -5,17 +5,23 @@ weight = 150
...
+#### [Mailing List](https://groups.google.com/forum/#!forum/submariner-dev)
+
+Join the developer mailing list.
 
-You can find a good guide about the github workflow here: [contributing to a github project](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project)
```
Comment (dfarrell07) [inline src/content/contributing/_index.en.md:27]:
```suggestion
```

File: src/content/contributing/_index.en.md [hunk 3]
```diff
@@ -5,17 +5,23 @@ weight = 150
...
 
-* Join the [developer mailing list](https://bit.ly/submariner-dev).
-* Join our [weekly meetings](https://docs.google.com/document/d/1qnZ2LpF_rXGfnYYPNTldQ4WbeEUxwnuQD-xTC6GbZdg/edit) and feel free to propose topics for discussion.
+### Talk to Us!
 
-### Pick up an issue to work on
+We would love to hear from you, how you are using Submariner, and what we can do to make it better. 
 
-### Or propose an enhancement via github issues
+#### [GitHub Project](https://github.com/submariner-io)
```
Comment (dfarrell07) [inline src/content/contributing/_index.en.md:13]:
```suggestion
#### [GitHub Project](https://github.com/submariner-io)

```

File: src/content/contributing/_index.en.md [hunk 4]
```diff
@@ -5,17 +5,23 @@ weight = 150
...
+Check out the project and consider contributing. Pick up an issue to work on or propose an enhancement by reporting a new issue; once your code is ready to be reviewed, you can propose a pull request. You can find a good guide about the GitHub workflow [here](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project).
 
-### Proposing a pull request
-Once your code is ready to be reviewed, you can propose a pull request.
+#### [#submariner](https://kubernetes.slack.com/archives/C010RJV694M)
```
Comment (dfarrell07) [inline src/content/contributing/_index.en.md:16]:
```suggestion
#### [#submariner](https://kubernetes.slack.com/archives/C010RJV694M)

```

File: src/content/contributing/_index.en.md [hunk 5]
```diff
@@ -5,17 +5,23 @@ weight = 150
...
+Share your ideas in the #submariner channel in Kubernetes' Slack. If you need it, you can [request an invite to K8S slack instance](https://slack.k8s.io/).
+
+#### [Community Calendar](https://calendar.google.com/calendar/r?cid=NHFuZGVoOGY0bzZ1ajlvZnBsczh1NWNlZ2tAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
```
Comment (dfarrell07) [inline src/content/contributing/_index.en.md:19]:
```suggestion
#### [Community Calendar](https://calendar.google.com/calendar/r?cid=NHFuZGVoOGY0bzZ1ajlvZnBsczh1NWNlZ2tAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)

```

File: src/content/contributing/_index.en.md [hunk 6]
```diff
@@ -5,17 +5,23 @@ weight = 150
...
+As a member of the Submariner Community, join any of our community meetings - no registration required. The weekly [Submariner Community Meeting](https://tinyurl.com/wfbx37q) (Tuesdays at 5:00pm CET) is a good place to start.
+
+
+#### [Mailing List](https://groups.google.com/forum/#!forum/submariner-dev)
+
+Join the developer mailing list.
 
-You can find a good guide about the github workflow here: [contributing to a github project](https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project)
```
Comment (mangelajo) [inline src/content/contributing/_index.en.md:20]: I'd keep this bit somewhere, I believe everybody is not aware of the github workflow, at least I wasn't

Comment (mangelajo) [reply src/content/contributing/_index.en.md:20]: Oh it's there sorry :-)

**Pull request merged**